src/lib.rs

```diff
@@ -3,7 +3,7 @@ use fxhash::{FxBuildHasher, FxHashMap};
 #[derive(Default)]
 pub struct Intern<'a> {
     data: FxHashMap<&'a str, InternId>,
-    list: Vec<String>,
+    list: Vec<Box<str>>,
```

Comment:
Technically we consume less memory right now, since Box<str> doesn't have capacity. So my question is why there isn't any specific performance impact with the proposed changes? I assume there is a difference, but it is not that significant compared to rest of the operations in the benchmark.

Reply:
I suspect it's just because it's such a small difference. Might be interesting to look at the assembly...